kubernetes-tests/pom.xml

```diff
@@ -76,6 +76,18 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>me.snowdrop</groupId>
+      <artifactId>servicecatalog-client</artifactId>
+      <version>1.1.1</version>
+    </dependency>
+    <dependency>
+      <groupId>me.snowdrop</groupId>
+      <artifactId>servicecatalog-mock</artifactId>
+      <version>1.1.1</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
As servicecatalog is a part of client now, you don't need to import it from snowdrop. Instead use : 
```
<groupId>io.fabric8</groupId>
<artifactId>servicecatalog-client</artifactId>
<version>${project.version}</version>
```

Reply:
Also, don't explicitly type version numbers. Use tags instead.